src/javascript/esutil.ts

```diff
@@ -251,16 +251,35 @@ export function toScannedMethod(
     }
 
     scannedMethod.params = (value.params || []).map((nodeParam) => {
-      let type = undefined;
-      let description = undefined;
-      // With ES6 we can have a lot of param patterns. Best to leave the
-      // formatting to escodegen.
-      let name = escodegen.generate(nodeParam);
-      // Rest parameters look like `...foo` in the parameter list, but are
-      // annotated as `@param {...T} foo` in the JSDoc.
-      if (name.startsWith('...')) {
-        name = name.substring(3);
+      let name;
+      let defaultValue;
+
+      if (nodeParam.type === 'Identifier') {
+        // Basic parameter: method(param)
+        name = nodeParam.name;
+
+      } else if (
+          nodeParam.type === 'RestElement' &&
+          nodeParam.argument.type === 'Identifier') {
+        // Rest parameter: method(...param)
+        name = nodeParam.argument.name;
+
+      } else if (
+          nodeParam.type === 'AssignmentPattern' &&
+          nodeParam.left.type === 'Identifier' &&
+          nodeParam.right.type === 'Literal') {
+        // Parameter with a default: method(param = "default")
+        name = nodeParam.left.name;
+        defaultValue = String(nodeParam.right.value);
```

Comment:
default values can be a lot of things. `[]`, `{}`, `new Set()` are all common and very reasonable values. I'd say just use escodegen to get a string representation

Reply:
Done.